### PR TITLE
code owners

### LINKED ISSUE #986: Complete code ownership
## Rationale

All the packages should be covered in `.github/CODEOWNERS`

### PR DESCRIPTION
**Pull request by ARR552**
Closes #986

### What does this PR does?

This PR adds the code owners.

### CODE REVIEW

**Comment by jeffprestes** [comment]
This is great. However it would be defined after Hermez-Node maintainers meeting in July 30th. Until there this PR will be freeze.

**Review by mfcastellani — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,25 @@
 *.md                    @jeffprestes
```
Comment (arnaubennassar) [inline .github/CODEOWNERS:1]:
```suggestion
*.md                    @jeffprestes @arnaubennassar
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
-db/l2db                 @arnaubennassar
-txselector/             @arnaubennassar
+api/                    @Mikelle @ARR552
+db/*.go                 @Mikelle @tclemos
+db/historydb            @Mikelle @tclemos
+db/migrations           @Mikelle @arnaubennassar
+db/l2db                 @Mikelle @marioidival
+db/kvdb                 @Mikelle @marioidival
+db/statedb              @Mikelle @marioidival
+txselector/             @mfcastellani @tclemos
```
Comment (arnaubennassar) [inline .github/CODEOWNERS:9]:
```suggestion
txselector/             @mfcastellani @tclemos @arnaubennassar
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,9 +1,25 @@
...
 etherscan/              @ARR552
-priceupdater/           @ARR552
+priceupdater/           @marioidival @ARR552
+common/                 @Mikelle @mfcastellani
+config/                 @ARR552 @marioidival
+coordinator/            @mfcastellani @marioidival
+node/                   @Mikelle @ARR552
+eth/                    @ARR552 @arnaubennassar
+cmd/                    @mfcastellani
+txprocessor/            @mfcastellani @tclemos
```
Comment (arnaubennassar) [inline .github/CODEOWNERS:18]:
```suggestion
txprocessor/            @mfcastellani @tclemos @arnaubennassar
```

**Review by arnaubennassar — CHANGES_REQUESTED** [review]

**Review by arnaubennassar — APPROVED** [review]

**Review by jeffprestes — APPROVED** [review]

**Review by Mikelle — APPROVED** [review]

**Pull request merged**